aten/src/ATen/native/native_functions.yaml

```diff
@@ -4906,7 +4906,7 @@
   use_c10_dispatcher: full
   variants: method, function
   dispatch:
-    CPU: legacy::cpu::_th_dist
+    CPU: dist
     CUDA: legacy::cuda::_th_dist
```

Comment:
Your implementation work both for cpu and cuda, is there a specific reason why you left the THC implementation for cuda?

Reply:
It's my first time to make PR for PyTorch, and reference to [Refactor philosophy](https://github.com/pytorch/pytorch/wiki/TH-to-ATen-porting-guide#refactor-philosophy) to do one thing a time. So I just change `dist` for CPU. After I finish the check of performance regression, I will update the code for cuda.

Reply:
Sure, no worries. I am fine if the final PR only contains the CPU update. My question was really to know if it was just out of scope of the PR or something else that prevented you from doing it.